app/services/order_movement.rb

```diff
@@ -1,25 +1,25 @@
 class OrderMovement
-  def initialize(fleet,quantity,destination)
+  def initialize(fleet, quantity, destination)
     @fleet = fleet
     @quantity = quantity
     @destination = destination
-    self.cancel_move! if @quantity == 0 || @destination == nil
+    cancel_move! if @quantity.zero? || @destination.nil?
   end
 
   def move!
```

Comment:
Prefer `if` over `unless` and extract these into smaller methods.

Reply:
fixed